Gems/PhysX/Code/Mocks/PhysX/MockSimulatedBody.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzFramework/Physics/Common/PhysicsSimulatedBody.h>
+#include <gmock/gmock.h>
+
+namespace UnitTest
+{
+    class MockSimulatedBody : public AzPhysics::SimulatedBody
```

Comment:
Same, there is one already defined here `Gems\ScriptCanvasPhysics\Code\Tests\ScriptCanvasPhysicsTest.cpp`. 

But since `SimulatedBody` is part of AzFramework it'd be nice if it'd be defined there and PhysX and ScriptCanvasPhysics just use it.

Reply:
I would have to get an approval for that from sig-core who owns AzFramework section. It would have to be a separate PR but it does sound good. I will file a ticket for it.

Reply:
No need to file a ticket for sig/core. `AzFramework/Physics` is owned by sig/simulation.